src/Arcus.Testing.Integration.DataFactory/DataFlowRunResult.cs

```diff
@@ -529,7 +529,7 @@ private DataFlowPreviewCsvTable(
             /// <summary>
             /// Loads the raw <paramref name="headerNames"/> and <paramref name="rows"/> to a validly parsed <see cref="CsvTable"/>.
```

Comment:
Just wondering if we should apply the following? 
```suggestion
            /// Loads the raw <paramref name="headerNames"/> and <paramref name="rows"/> to a validly parsed <see cref="DataFlowPreviewCsvTable"/>.
```

Reply:
Same for me, it's 8nternal anyway. Applied 👍